metrics/web/test/base/presentation/graphs/animated_bar_test.dart

```diff
@@ -0,0 +1,182 @@
+// Use of this source code is governed by the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:metrics/base/presentation/graphs/animated_bar.dart';
+import 'package:metrics/base/presentation/widgets/rive_animation.dart';
+import 'package:rive/rive.dart';
+
+import '../../../test_utils/test_animation_container.dart';
+
+// ignore_for_file: avoid_redundant_argument_values
+
+void main() {
+  group("AnimatedBar", () {
+    const asset = 'asset';
+    final riveAnimationFinder = find.byType(RiveAnimation);
+
+    testWidgets(
+      "throws an AssertionError if the given height is null",
+      (WidgetTester tester) async {
+        await tester.pumpWidget(
+          const _AnimatedBarTestbed(
+            height: null,
+          ),
+        );
+
+        expect(tester.takeException(), isAssertionError);
+      },
+    );
+
+    testWidgets(
+      "throws an AssertionError if the given height is negative",
+      (WidgetTester tester) async {
+        await tester.pumpWidget(
+          const _AnimatedBarTestbed(
+            height: -1.0,
+          ),
+        );
+
+        expect(tester.takeException(), isAssertionError);
+      },
+    );
+
+    testWidgets(
+      "throws an AssertionError if the given width is null",
+      (WidgetTester tester) async {
+        await tester.pumpWidget(
+          const _AnimatedBarTestbed(
+            width: null,
+          ),
+        );
+
+        expect(tester.takeException(), isAssertionError);
+      },
+    );
+
+    testWidgets(
+      "throws an AssertionError if the given width is negative",
+      (WidgetTester tester) async {
+        await tester.pumpWidget(
+          const _AnimatedBarTestbed(
+            width: -1.0,
+          ),
+        );
+
+        expect(tester.takeException(), isAssertionError);
+      },
+    );
+
+    testWidgets(
+      "displays the rive animation with the given asset",
```

Comment:
```suggestion
      "applies the given asset to the rive animation widget",
```
Would this description describe this test a bit better?